ansible/install-apps.yml

```
@@ -0,0 +1,33 @@
+
```

Why did you remove the document separator? Add it back
```suggestion
---
```